services/src/floodlight-modules/src/main/java/org/openkilda/floodlight/kafka/RecordHandler.java

```diff
@@ -197,6 +200,7 @@ private Destination getDestinationForTopic(String replyToTopic) {
     }
 
     private void doDiscoverIslCommand(CommandMessage message) {
+        logger.info("Discovering ISL");
```

Comment:
It will be extremely verbose - thousands requests are goes here. It have it's own specific logging mechanism inside PathVerificationService, so this message must be removed.